src/containers/AppBody.jsx

```diff
@@ -17,7 +18,8 @@ const styles = {
 class AppBody extends Component {
   componentDidMount() {
     const { dispatch, snippetKey } = this.props;
-    axios.get("../snippet.json")
+    const [user, snippet] = snippetKey.split("/");
```

Comment:
It could be verified when runestone compiles, although that isn't yet implemented.